pkg/vm/vm.go

```diff
@@ -1458,9 +1458,16 @@ func (v *VM) Jump(ctx *Context, offset int) {
 	ctx.nextip = offset
 }
 
-// Call calls method by offset. It is similar to Jump but also
-// pushes new context to the invocation state
-func (v *VM) Call(ctx *Context, offset int) {
+// CallAndIncrementInvocations calls method by offset. It is similar to Jump but also
+// pushes new context to the invocation state and increments invocation counter
```

Comment:
to the evaluation stack?

Reply:
To the invocation stack. Fixed typo: `state` -> `stack`.